file-formats/cota/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -0,0 +1,80 @@
+INTERFACE zif_aff_cota_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Connection Target Object</p>
+    "! Creation of connection target object
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version        TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Connection Target Object ID</p>
+      "! Connection target object ID
+      conn_id               TYPE c length 30,
+      "! <p class="shorttext">Version</p>
+      "! Version
+      version               TYPE r3state,
+      "! <p class="shorttext">Created By</p>
+      "! Created by
+      createdby             TYPE as4user,
+      "! <p class="shorttext">Created On</p>
+      "! Created on
+      createdon             TYPE as4date,
+      "! <p class="shorttext">Creation Date</p>
+      "! Creation date
+      createdat             TYPE as4time,
+      "! <p class="shorttext">Changed By</p>
+      "! Changed By
+      changedby             TYPE as4user,
+      "! <p class="shorttext">Changed On</p>
+      "! Changed On
+      changedon             TYPE as4date,
+      "! <p class="shorttext">Change Date</p>
+      "! Change date
+      changedat             TYPE as4time,
```

Comment:
These fields should not be part of the file format.

See also https://github.com/SAP/abap-file-formats/blob/main/README.md#background-and-scope

> The files don't contain metadata like dates, usernames, last editor, system specific information, plus no obsolete or redundant information as well as no autogenerated content.